main/src/main/scala/sbt/internal/CommandExchange.scala

```diff
@@ -106,15 +113,18 @@ private[sbt] final class CommandExchange {
       case Some(x) => x
       case None    => ConnectionType.Tcp
     }
-    val serverLogLevel: Level.Value = Level.Debug
+    lazy val level: Level.Value = (s get serverLogLevel)
```

Comment:
shouldn't this be `s get (logLevel in serverLogLevel)`?

Reply:
As discussed

```scala
.setCond(serverLogLevel, srvLogLevel)
```

in Project.scala where srvLogLevel does the lookup.